blazor/dropdown-list/selection.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: Selection in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Selection feature in Syncfusion Blazor DropDownList component and more.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# Selection
+
+## Get Selected Value

```

Comment:
In h2 heading starting word staring letter should be caps, other words start with small case.